### PR TITLE
implement translation api

### PR DESCRIPTION
**Pull request by Andrewmob2703-zz**
add language select in contact_me.html
translate comment to language code submitted by form

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -123,9 +134,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       finalResponse = "Got no comment.";
       response.getWriter().println("<h3>" + finalResponse + "</h3>");
     } else {
+      System.out.println(languageCode);
```
Comment (peizhaoou) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:138]: Just a general comment. A console print statement is critical to catching bug, so the context is very important, especially when you work on a codebase with hundreds of thousands lines of code (or even more). E.g., consider
  System.out.println("DataServlet/doPost: language code = " + languageCode);

This is just an example to illustrate the point, you could give more context in your preferred way.

**Review by peizhaoou — APPROVED** [review]
LGTM. Great progress:)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (Andrewmob2703-zz) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:138]: oh! I forgot to take that out. I didn't mean to include that.
But thanks

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -101,13 +104,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String comment = getParameter(request, COMMENT, "");
+    String languageCode = getParameter(request, "lang", "");
     String user = getParameter(request, NAME, "");
     String userEmail = userService.getCurrentUser().getEmail();
     long timestamp = System.currentTimeMillis();
+
+    // Translate comment.
```
Comment (martiKallas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:112]: nit: you probably don't need so many comments. Your use of variable names and the functions makes this "self-documenting." A person can pretty reasonably understand what is going on without needing a comment. Typically, we use comments for describing how functions and classes behave, to describe the why of complicated code, or to explain why we did something that is normally considered bad design.

You can read more here: https://google.github.io/styleguide/cppguide.html#Comments

**Review by martiKallas — APPROVED** [review]
Nice!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (Andrewmob2703-zz) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:112]: Great, thanks

**Pull request merged**